src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_6001_7000.cpp

```diff
@@ -6694,6 +6695,26 @@ GUI_TEST_CLASS_DEFINITION(test_6979) {
     CHECK_SET_ERR(tabWidget->currentIndex() == 1, "Expected second tab to be active")
 }
 
+GUI_TEST_CLASS_DEFINITION(test_6981) {
+    // Open murine.gb
+    GTFileDialog::openFile(os, dataDir + "samples/Genbank/murine.gb");
+    GTUtilsSequenceView::checkSequenceViewWindowIsActive(os);
+
+    // Create an annotation Build phy-tree with default settings 5809..5809
```

Comment:
There is something wrong in the comment.

Reply:
Fixed